### PR TITLE
Adds tracks list in album. #82

### PR DESCRIPTION
**Pull request by hayribakici**
This PR adds tracks into the album model and solves the issue #82.

### CODE REVIEW

**Review by roisnir — DISMISSED** [review]
looks good

**Comment by hayribakici** [comment]
@roisnir @rinukkusu Can I merge it?

File: lib/src/models/album.dart [hunk 1]
```diff
@@ -97,4 +97,12 @@ class AlbumSimple extends Object {
...
   /// The Spotify URI for the album.
   String uri;
+
+  /// The tracks of this album wrapped in a page object.
+  @JsonKey(name: 'tracks')
+  Paging<TrackSimple> tracksPage;
+
+  Iterable<TrackSimple> get tracksNative =>
+      tracksPage?.itemsNative?.map((e) => TrackSimple.fromJson(e))?.toList() ??
+      [];
```
Comment (roisnir) [inline lib/src/models/album.dart:107]: In a second thought, maybe the right place for something like this is in src/endpoints/tracks.dart using  `Pages.fromPaging`
Something like
`src/endpoints/tracks.dart:`
```dart
Pages<TrackSimple> fromPaging(Paging<TrackSimple> paging) => 
    Pages<TrackSimple>.fromPaging(_api, paging, (json) => TrackSimple.fromJson(json));

```

Comment (hayribakici) [reply lib/src/models/album.dart:107]: This is a good idea. I'll see what I can do. Thank you!

**Review by rinukkusu — APPROVED** [review]
LGTM 👍

**Pull request merged**